slash/loader.py

```diff
@@ -108,8 +109,9 @@ def _address_in_file_matches(self, address_in_file, test):
         test_address_in_file = test.__slash__.address_in_file
```

Comment:
I'm not sure you need this. You will get a variation with a proper id here, and you can compare against your saved id... Why do you need to parse the variation from the string?